src/simplicial-hott/07-discrete.rzk.md

```diff
@@ -466,17 +466,20 @@ Discrete types are automatically Segal types.
     ( first
       ( equiv-square-sigma-over-product extext A is-discrete-A x y z w f g)
       ( p , (q , τ))) =
-    ( square-sigma-over-product extext A is-discrete-A x y z w f g (p , (q , τ)))
+    ( square-sigma-over-product
+        extext A is-discrete-A x y z w f g (p , (q , τ)))
   :=
     idJ
     ( A ,
       y ,
       \ w' q' ->
-        (g : hom A z w') ->
-        (τ : product-transport A A (hom A) x z y w' p q' f = g) ->
-        (first (equiv-square-sigma-over-product extext A is-discrete-A x y z w' f g))
-          (p , (q' , τ)) =
-        (square-sigma-over-product extext A is-discrete-A x y z w' f g) (p , (q' , τ)) ,
+      ( g : hom A z w') ->
+      ( τ : product-transport A A (hom A) x z y w' p q' f = g) ->
+      ( first (equiv-square-sigma-over-product
+                extext A is-discrete-A x y z w' f g))
```

Comment:
When breaking an expression into multiple lines, the convention is to break at the earliest possible branching point. This is to avoid excessive indentation. E.g. here
```rzk
      ( first
        ( equiv-square-sigma-over-product extext A is-discrete-A x y z w' f g))
```
or
```rzk
      ( first
        ( equiv-square-sigma-over-product
            extext A is-discrete-A x y z w' f g))
```